src/validate/methods/arrayOfStrings.ts

```diff
@@ -0,0 +1,20 @@
+export const arrayOfStrings = (value: string) => {
+  if (typeof value !== "string") {
+    return "Expected strings of array.";
+  }
+
+  try {
+    const parsedValue = JSON.parse(value);
+
+    if (Array.isArray(parsedValue)) {
+      for (const item of parsedValue) {
+        if (typeof item !== "string") {
+          return "Expected a string.";
```

Comment:
Would this message make the error more clear?

```suggestion
          return "All values in the array must be a string.";
```